Makefile

```diff
@@ -3,47 +3,54 @@
 PREFIX := /usr
 DESTDIR :=
 
-ifndef VERSION
-MAJORVERSION := 8
-MINORVERSION ?= $(shell git rev-list --count master)
-endif
-VERSION := ${MAJORVERSION}.${MINORVERSION}
+
+MAJORVERSION$(VERSION) := 8
+MINORVERSION$(VERSION) != git rev-list --count master
+VERSION ?= ${MAJORVERSION}.${MINORVERSION}
 
 LDFLAGS := -ldflags '-s -w -X main.version=${VERSION}'
 ARCH := $(shell uname -m)
 PKGNAME := yay
 BINNAME := yay
 PACKAGE := ${PKGNAME}_${VERSION}_${ARCH}
 
-export GOPATH=$(shell pwd)/.go
+GOPATH=$(shell pwd)/.go
 
 default: build
 
+.PHONY: all
```

Comment:
Setting the various rules that don't correspond to files to `.PHONY` means that make won't be trying to check if ifles with that name exist which could cause rules to not be run (eg. if I did `touch all` for some reason, the rule would fail before).